src/Nethermind/Nethermind.Monitoring/Metrics/MetricsController.cs

```diff
@@ -24,6 +25,7 @@ public partial class MetricsController : IMetricsController
         private Timer _timer;
         private readonly Dictionary<Type, (MemberInfo, string, Func<double>)[]> _membersCache = new();
         private readonly Dictionary<Type, (string DictName, IDictionary<string, long> Dict)> _dynamicPropCache = new();
+        private readonly Dictionary<Type, (MemberInfo, string GaugeName, string LabelName, IDictionary)[]> _labelledDictionaryCache = new();
```

Comment:
Can we unify `_dynamicPropCache` and `_labelledDictionaryCache` and just have empty labels or label same as key (or whatever)? Wouldn't it make the code simpler?

Reply:
Done